development/gcp/pkg/secretmanager/client.go

```diff
@@ -0,0 +1,65 @@
+package secretmanager
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"google.golang.org/api/option"
+	gcpsecretmanager "google.golang.org/api/secretmanager/v1"
+)
+
+// NewService creates new Service struct
+func NewService(ctx context.Context, options ...option.ClientOption) (*Service, error) {
+	secretManagerClient, err := gcpsecretmanager.NewService(ctx, options...)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create google Secret Manager client, got error: %w", err)
+	}
+	secretVersionsService := gcpsecretmanager.NewProjectsSecretsVersionsService(secretManagerClient)
+
+	return &Service{Service: secretManagerClient, VersionService: secretVersionsService}, nil
+}
+
+// AddSecretVersion adds a new version to a secret
+// expects secretPath in "projects/*/secrets/*" format
+func (sm *Service) AddSecretVersion(secretPath string, secretData []byte) (*gcpsecretmanager.SecretVersion, error) {
+	newVersionRequest := gcpsecretmanager.AddSecretVersionRequest{Payload: &gcpsecretmanager.SecretPayload{Data: base64.StdEncoding.EncodeToString(secretData)}}
+	newVersionCall := sm.Service.Projects.Secrets.AddVersion(secretPath, &newVersionRequest)
+	secretVersion, err := newVersionCall.Do()
+	return secretVersion, err
```

Comment:
You could simplify this into one line without adding assignment. That suggestion applies to the similar cases in the file.

```suggestion
	return sm.Service.Projects.Secrets.AddVersion(secretPath, &newVersionRequest).Do()
```